### PR TITLE
[Blocks Editor] Description and placeholder added

### PR DESCRIPTION
**Pull request by madhurisandbhor**
### What does it do?

Description and placeholder added for Blocks editor

### Why is it needed?

CTB configure the view option doesn't work

### How to test it?

In CTB , Set some placeholder and description value for the blocks type and check if its reflected in CM.

### CODE REVIEW

**Comment by remidej** [comment]
Hey sorry for not reviewing this quickly. FYI you have a conflict now

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js [hunk 1]
```diff
@@ -13,12 +13,14 @@ jest.mock('@strapi/helper-plugin', () => ({
...
   render(
     <Blocks
       intlLabel={{ id: 'blocks', defaultMessage: 'blocks type' }}
       name="blocks-editor"
-      value={blocksData}
+      value={value}
+      hint="blocks description"
+      placeholder={{ id: 'blocksPlaceholder', defaultMessage: 'blocks placeholder' }}
       onChange={jest.fn()}
       {...props}
```
Comment (remidej) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js:25]: I don't think you need to treat value differently, spreading props handles it

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js [hunk 2]
```diff
@@ -34,20 +36,24 @@ const setup = (props) =>
...
-  it('should render blocks without error', () => {
-    setup();
+  it('should render blocks without error', async () => {
+    setup({ value: null });
 
     expect(screen.getByText('blocks type')).toBeInTheDocument();
+    expect(screen.getByText('blocks description')).toBeInTheDocument();
+    await waitFor(() => {
+      expect(screen.getByText('blocks placeholder')).toBeInTheDocument();
+    });
```
Comment (remidej) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js:46]: I'm a bit surprised that waitFor is necessary, do you know why?

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js (hunk 1 above)
Comment (madhurisandbhor) [reply packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js:25]: yeahh 👍

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js (hunk 2 above)
Comment (madhurisandbhor) [reply packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js:46]: actually I struggled to locate placeholder for quite sometime, but it appears placeholder gets updated in DOM quite late.

**Review by remidej — APPROVED** [review]
Tested ✅

**Pull request merged**